### DISCUSSION TITLE
【求助】我的帮布掉空洞里了！

### DISCUSSION THREAD

**Discussion by naichatiandian** [share121/inter-knot#1895] [其他]
事情是这样子的，前天我收到了封邮件，邮件里说3号空洞内有免费的攒劲歌舞表演，昨天我和我的帮布就去了没想到是骗局！一帮匪徒抢走了我的帮布！费了九牛二虎之力我才跑了出来，有没有绳匠能救救我的帮布他是我的兄弟啊！！！必有重谢！！！

**Comment by 834398220** [comment]
这个委托我接了
